### DISCUSSION TITLE
IREGADJ=1 in PEST_HP unexpectedly behaving like IREGADJ=2

### DISCUSSION THREAD

**Discussion by sjepsen395** [usgs/pestpp#310] [Q&A]
Hi All,

This question pertains to PEST_HP, so this may not be the best venue. But I'll try anyways. I've been doing some testing to understand PEST_HP outputs for different regularization and prior information settings. I'm finding that IREGADJ=1 is not working as expected based on PEST documentation. When I have IREGADJ=1, PEST_HP automatically calculates the weight of each prior information equation such that the sum of weights over each regularization group is the same, with relative within-group weight proportions maintained. This is what IREGADJ=2 is supposed to do, if I understand correctly. Does anyone know if there is a setting in one's control file that could cause IREGADJ=1 to inadvertently behave like IREGADJ=2? I wonder if this could be a bug.

Thanks,
Steve

**Comment by doherty1234** [comment]
Hello Steve

I'd have to look into this some more to answer your questions. Unfortunately I am travelling now, so I will have to postpone this until I get home.

Thanks for the information.

John Doherty

>

**Reply by sjepsen395** [reply]
Hi John,

Great, thanks for looking at this.

With IREGADJ set to 1, I was able to reproduce PEST_HP outputs of regularization "phi" and objective functions from output parameter values and the assumptions of IREGADJ=2 as I understood them.

Steve

**Comment by sjepsen395** [comment]
All,

I have an update on a set of runs I did to look into my original post, above. I got identical PEST_HP outputs using IREGADJ settings of 1, 2, and 3. My prior information weights were set to 1.0 in all runs. The values of regularization weights printed to the *.rec file reflect an IREGADJ setting of 2 according to the PEST User Manual Part 1. My TextPad search found no mention of setting "IREGADJ" in the *.rec output file from PEST_HP.

Steve
